calibrators/marker_radar_lidar_calibrator/launch/calibrator.launch.xml

```diff
@@ -4,9 +4,11 @@
   <arg name="calibration_service_name" default="calibrate_radar_lidar"/>
   <arg name="rviz" default="true"/>
   <let name="rviz_profile" value="$(find-pkg-share marker_radar_lidar_calibrator)/rviz/default.rviz"/>
-  <arg name="radar_parallel_frame" default="front_unit_base_link"/>
+  <arg name="radar_optimization_frame" default="base_link"/>
   <arg name="input_lidar_pointcloud" default="/sensing/lidar/front_lower/pointcloud_raw"/>
-  <arg name="input_radar_objects" default="/sensing/radar/front_center/objects_raw"/>
+  <arg name="input_radar_msg" default="/sensing/radar/front_center/objects_raw"/>
+  <arg name="msg_type" default="objects_raw"/>
```

Comment:
for msg_type and transformation_type, please provide alternatives with the choice attribute

Reply:
Fixed in the new PR https://github.com/tier4/CalibrationTools/pull/221